sfs/array.py

```diff
@@ -423,16 +460,38 @@ def cube(N, spacing, center=[0, 0, 0], orientation=[1, 0, 0]):
 
 
 def sphere_load(fname, radius, center=[0, 0, 0]):
-    """Spherical secondary source distribution loaded from datafile.
+    """Spherical secondary source distribution loaded from the datafile fname.
 
-    ASCII Format (see MATLAB SFS Toolbox) with 4 numbers (3 position, 1
-    weight) per secondary source located on the unit circle.
+    ASCII Format (see MATLAB SFS Toolbox) with 4 numbers (3 for the cartesian 
+    position vector, 1 for the integration weight) per secondary source located 
+    on the unit circle which is resized by the set radius and shifted to the
```

Comment:
`*radius*` and `*center*`.

I think "given" would be better than "set", but I'm no native speaker ...

Reply:
changed to "given"